game/ffmpeg.cc

```diff
@@ -138,14 +138,22 @@ void FFmpeg::operator()() {
 	m_duration = m_formatContext->duration / double(AV_TIME_BASE);
 	audioQueue.setDuration(m_duration);
 	int errors = 0;
+	bool eof = false;
 	while (!m_quit) {
+		if(eof) {
+			// FIXME: use a boost::condition on m_quit
+			boost::thread::sleep(now() + 0.1);
+			continue;
+		}
 		try {
 			if (audioQueue.wantSeek()) m_seekTarget = 0.0;
 			if (m_seekTarget == m_seekTarget) seek_internal();
 			decodePacket();
 			errors = 0;
 		} catch (eof_error&) {
 			videoQueue.push(new Bitmap()); // EOF marker
+			eof = true;
+			std::clog << "ffmpeg/debug: done loading " << m_filename << std::endl;
 			boost::thread::sleep(now() + 0.1);
```

Comment:
I'd rewrite this with 
`std::this_thread::sleep_for(chrono::milliseconds(100));`

also include the header files: 
```
#include <thread>
#include <chrono>
```

Reply:
Removed (already in the while loop